LayoutTests/platform/mac-wk2/TestExpectations

```diff
@@ -1592,7 +1592,7 @@ webkit.org/b/261356 [ Debug ] fast/mediastream/device-change-event-2.html [ Pass
 webkit.org/b/263396 css3/color/text.html [ Skip ]
 
 # WebGPU
-[ Release ] http/tests/webgpu/webgpu/api/validation/encoding/cmds/copyTextureToTexture.html
+[ Release ] http/tests/webgpu/webgpu/api/validation/encoding/cmds
```

Comment:
I think this should say:
```
[ Release ] http/tests/webgpu/webgpu/api/validation/encoding/cmds [ Pass ]
```
at least locally I need to add `[ Pass ]` to avoid the warning about test passed but expected to fail